app/src/main/java/org/simple/clinic/di/network/NetworkModule.kt

```diff
@@ -13,6 +13,8 @@ import org.simple.clinic.drugs.search.DrugCategory
 import org.simple.clinic.drugs.search.DrugFrequency
 import org.simple.clinic.medicalhistory.Answer
 import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireType
+import org.simple.clinic.monthlyReports.questionnaire.component.properties.InputFieldType
```

Comment:
Drop `monthlyReports` from the namespace path across entire codebase

Reply:
`monthlyReports` package is to distinguish from other part of the app which uses the same server side rendering technique.